src/txn/txn_ckpt.c

```diff
@@ -138,18 +140,27 @@ __checkpoint_flush_tier(WT_SESSION_IMPL *session, bool force)
                 }
             }
             /* Only instantiate the handle if we need to flush. */
-            WT_ERR(__wt_session_get_dhandle(session, key, NULL, NULL, 0));
+            ret = __wt_session_get_dhandle(session, key, NULL, NULL, 0);
+            /* If we get back EBUSY, this handle may be open with bulk or other special flags. We
+             * need to skip it this tree. We fake checkpoints for them, i.e. we never really write
```

Comment:
Is there a typo? Did you mean
```suggestion
             * need to skip this tree. We fake checkpoints for them, i.e. we never really write
```
Or
```suggestion
             * need to skip it for this tree. We fake checkpoints for them, i.e. we never really write
```